bezpy/mt/io.py

```diff
@@ -189,3 +192,29 @@ def get_1d_site(name):
         return _SITES1D[newname]
 
     raise ValueError("No 1d site profile with the name: " + name)
+
+def read_runinfo(site,root):
+    site.runinfo = {}
+
+    # fieldnotes = xml_site.find("FieldNotes")
+    for field in root.findall("FieldNotes"):
+        # runid of fieldnote
+        runid = field.attrib['run']
+        site.runinfo[runid] = {}
+
+        try:
+            site.NIMSid = get_text( field.find('Instrument'), 'Id')
```

Comment:
Could there be a different NIMSid for different runs? If so, would it make sense to move the NIMSid into the `runinfo` dictionary? If there is only one NIMSid, I'd suggest moving this up to the `read_xml` function and updating the `site` object there instead.

Reply:
That's a good point. That rarely happens, but one of the field crew told me that can happen when there's an unknown error in the previous run. I've updated the code. This code is relating to the newly developed code. I'll fix these new parts too.